### ISSUE TITLE
[Errno 8] nodename nor servname provided, or not known

### ISSUE THREAD

**Issue by tasinttttttt** [vikstrous/pirate-get#148]
Here's the complete error:

```bash
Trying https://apibay.org... Failed (HTTP Error 521: )
Could not fetch mirrors :(
[Errno 8] nodename nor servname provided, or not known
```

**Comment by rnhmjoj** [comment]
The first error (HTTP 521) means "server is down" according to wikipedia. I've just received it too a couple of times too: I guess tpb is having some issues.

For the second, it seems proxybay.bz no longer exists.
I'll have to find a new mirror list.

Btw, you can specify a mirror manually with `-m`.

**Comment by rnhmjoj** [comment]
I just pushed an update to fix this (0.4.2).
